sdk/src/connections/network/network_depth_sensor.cpp

```diff
@@ -85,6 +86,7 @@ NetworkDepthSensor::~NetworkDepthSensor() {
 aditof::Status NetworkDepthSensor::open() {
     using namespace aditof;
 
+    std::string connectionString;
```

Comment:
This doesn't seem to be used anywhere, nor the connection_validator.h.

Reply:
removed it.